pkg/question/question.go

```diff
@@ -347,7 +347,7 @@ func AskIfEnterInstanceType(h *ec2helper.EC2Helper, defaultInstanceType string)
 	data := [][]string{{"Enter the instance type"}, {"Provide vCPUs and memory information for advice"},
 		{fmt.Sprintf("Use the default instance type, [%s]", *defaultOption)}}
 	indexedOptions := []string{cli.ResponseYes, cli.ResponseNo, *defaultOption}
-	question := "Instance Select Method"
+	question := "Select the method for instance selection:"
```

Comment:
This question could use a little more clarity. I'd suggest

```suggestion
	question := "How do you want to choose the instance type?"
```